### ISSUE TITLE
training loss: nan

### ISSUE THREAD

**Issue by croakthunder** [Tramac/DenseVoxNet-tensorflow#2]
Hi, when I am working on this model, the training loss is 'nan' since the beginning. 
Step: 0, Train loss: nan, Time: 9.87615s
Step: 1, Train loss: nan, Time: 1.97912s
Step: 2, Train loss: nan, Time: 1.98144s
Step: 3, Train loss: nan, Time: 1.9938s
I use just follow the Read Me file, have you encountered this situation? Hope for your reply, thanks a lot.

**Comment by Tramac** [comment]
Hi, you can try to change the parameter `batch_size=8`, `learning_rate=0.001` in `train.py`.
